README.md

```diff
@@ -1,47 +1,178 @@
 # Pulumi Native Provider Boilerplate
 
-This repo is a boilerplate showing how to create a native Pulumi provider.  
+This repository is a boilerplate showing how to create a native Pulumi provider.
 
 ### Background
 
 This repository is part of the [guide for authoring and publishing a Pulumi Package](https://www.pulumi.com/docs/guides/pulumi-packages/how-to-author).
 
 Learn about the concepts behind [Pulumi Packages](https://www.pulumi.com/docs/guides/pulumi-packages/#pulumi-packages).
 
-## Creating a Pulumi Native Provider
+See where Resource Providers fit into [the Pulumi architecture](https://www.pulumi.com/docs/intro/concepts/how-pulumi-works/#how-pulumi-works)
```

Comment:
What are we trying to say with this line?

Reply:
A reminder of the architecture diagram may help those more visually inclined. Changing the wording to clarify.